src/platform_impl/windows/dark_mode.rs

```diff
@@ -54,8 +54,8 @@ lazy_static! {
         }
     };
 
-    static ref DARK_THEME_NAME: Vec<u16> = widestring("DarkMode_Explorer");
-    static ref LIGHT_THEME_NAME: Vec<u16> = widestring("");
+    static ref DARK_THEME_NAME: Vec<u16> = OsStr::new("DarkMode_Explorer").encode_wide().chain(once(0)).collect();
+    static ref LIGHT_THEME_NAME: Vec<u16> = OsStr::new("").encode_wide().chain(once(0)).collect();
```

Comment:
This is exactly what windows-rs does inside its `IntoParam<'a, PWSTR> for &OsStr` implementation - can't you just pass those as-is or store the resulting `Param::Boxed` here?